docs/source/_templates/sdk_TP_tutorial.rst

```diff
@@ -12,55 +12,18 @@
     {% set lowercase_lang = 'go' %}
 {% endif %}
 
-***********************************************
-Transaction Processor Tutorial  {{ language }}
-***********************************************
+*****************************************************
+Transaction Processor: Creating a Transaction Handler
+*****************************************************
 
-Overview
-========
+A transaction processor has two top-level components: a processor class and
```

Comment:
With calling out the processor class, I would mention that the processor class is completely supplied by the sdk, whereas the handler class is written by the transaction family writer.